spec/order_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'order'
+
+describe Order do
+
+	let (:order){double(:order)}
```

Comment:
Also, tabs! Don't forget to make sure your text editor uses spaces instead of tabs.